src/controller/proxy/controller.go

```diff
@@ -94,22 +92,44 @@ func (c *controller) UseLocalBlob(ctx context.Context, art lib.ArtifactInfo) boo
 	return exist
 }
 
-func (c *controller) UseLocalManifest(ctx context.Context, art lib.ArtifactInfo) bool {
-	if len(art.Digest) == 0 {
-		return false
+func (c *controller) UseLocalManifest(ctx context.Context, art lib.ArtifactInfo, remote RemoteInterface) (bool, error) {
```

Comment:
Why in `ProxyManifest` the `remote` is created inside the func but in this case the `remote` is sent by parm

Reply:
Updated, remove this param